src/item/components/modals/AddToCollectionModal.tsx

```diff
@@ -76,7 +76,15 @@ const AddToCollectionModal: FunctionComponent<AddToCollectionModalProps & UserPr
 				[],
 				ContentTypeNumber.collection,
 				undefined,
-				undefined
+				undefined,
+				[
+					{
+						contributors: {
+							profile_id: { _eq: getProfileId(user) },
+							rights: { _eq: 'CONTRIBUTOR' },
+						},
+					},
+				]
```

Comment:
shouldn't this contain another _or statement where we specify, it has to be the owner or a contributor with CONTRIBUTOR rights?
since the title of the function seems to suggest that's what we're looking for.
```suggestion
				undefined,
				[
					{
						_or: [
              						{
                						contributors: {
									profile_id: { _eq: getProfileId(user) },
									rights: { _eq: CONTRIBUTOR }
            							}
              						},
              						{
                						owner_profile_id: {_eq:  getProfileId(user) }
              						}
            					]
					}
				]
```

Reply:
Helaas niet het gewenste resultaat. Zie https://github.com/viaacode/avo2-client/pull/1945/commits/d974e77273a15dacd57c832ae7264def6b89e678